AWS_FLB_CHERRY_PICKS

```diff
@@ -33,3 +33,6 @@ https://github.com/matthewfala/fluent-bit.git datadog-ecs-patch 3c1ad69ada5bb6f2
 
 # S3 retry limit configuration option -- Cherrypick from combined branch that resolves s3 merge conflict
 https://github.com/matthewfala/fluent-bit.git 2.31.1-all-cherrypicks 26c0b0c779b1747a1c751c05d96a597e443bf29a
+
+# CloudWatch_Logs sync core scheduler fix
```

Comment:
I think it should match up with the changelog entry and say "duplicate tag match fix in sync core scheduler" something like that